0.10/Dockerfile.rhel7

```diff
@@ -32,6 +35,8 @@ COPY ./s2i/bin/ $STI_SCRIPTS_PATH
 # run and build the applications.
 COPY ./contrib/ /opt/app-root
 
+RUN $PROMPT_FILE && npm install -g nodemon
```

Comment:
(his team should also review this PR as they are the maintainers of this image going forward)